Makefile

```diff
@@ -41,250 +25,224 @@ COMMON_ARGS += --build-arg=TOOLS=$(TOOLS)
 COMMON_ARGS += --build-arg=SHA=$(SHA)
 COMMON_ARGS += --build-arg=TAG=$(TAG)
 COMMON_ARGS += --build-arg=GO_VERSION=$(GO_VERSION)
-COMMON_ARGS += .
+COMMON_ARGS += --build-arg=IMAGES=$(ARTIFACTS)/images
 
-TESTPKGS ?= ./...
+all: help
+
+# Help Menu
+
+define HELP_MENU_HEADER
```

Comment:
Take a close look here. This is the Makefile help banner that should have enough to get someone started developing Talos.